README.md

```diff
@@ -115,3 +115,11 @@ OpenZeppelin exists thanks to its contributors. There are many ways you can part
 ## License
 
 OpenZeppelin is released under the [MIT License](LICENSE).
+
+[API docs]: https://docs.openzeppelin.org/v2.3.0/api/token/erc721
```

Comment:
```suggestion
[API docs]: https://docs.openzeppelin.org/v2.3.0/api/token/erc20
```

The ERC721 docs are not that great yet, I think the ERC20 ones are more complete

Reply:
It's a shame that we can't have a link just to the api. e.g. https://docs.openzeppelin.org/v2.3.0/api

It would also be nice to just use latest where appropriate.  e.g. https://docs.openzeppelin.org/latest/api

Finally, openzeppelin-eth is at v2.1.3 so would ideally refer to version 2.1.3 of the docs (assuming this is correct) but this link is https://5ca2514b224a4200080bb4c3--openzeppelin-docs2.netlify.com/docs/2.1.3/token_erc20_erc20

Reply:
These are all interesting points, @frangio and I are still in the process of figuring out what will work best as a versioning solution.

Thanks!